### PR TITLE
integrate homology dump steps into rr ftp pipeline

### PR DESCRIPTION
**Pull request by vinay-ebi**
## Description

Integrate Homology dump steps into RR ftp dump pipeline

## Benefits

Can run homoly dumps alone for any given division or species

### CODE REVIEW

**Comment by vinay-ebi** [comment]
we can ignore the build failure related to licences those will be fixed during 110 merge to main 
Failed test '/home/travis/build/Ensembl/ensembl-production/modules/t/../../scripts/misc/tmux_ls.sh does not have an Apache v2.0 licence declaration and correct Copyright year [2016-2023] in the first 30 lines'

File: modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm [hunk 1]
```diff
@@ -76,9 +76,20 @@ sub default_options {
         gff3_per_chromosome  => $self->o('per_chromosome'),
         gtf_per_chromosome   => $self->o('per_chromosome'),
         xref_external_dbs    => [],
+	dump_homologies_script => $self->o('ENV','ENSEMBL_ROOT_DIR') . "/ensembl-compara/scripts/dumps/dump_homologies.py",
+	rr_version => $self->o('ENV', 'RR_VERSION'),
+	ref_dbname => 'ensembl_compara_references',
+	homology_dump_server_url => $self->get_server_url('st5'),
```
[the remark this answers is not in the document]
Comment (marcoooo) [reply modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm:82]: Just realised one thing, this pipeline won't work if the user doesn't have the shortcut setup in his environment, which is possible. Can't we just add the output of `$host_name details url` to the parameters? Simpler and more portable.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm (hunk 1 above)
[the remark this answers is not in the document]
Comment (vinay-ebi) [reply modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm:82]: yes we getting host details  : https://github.com/Ensembl/ensembl-production/pull/774/files#diff-7ee5c751c6f2c94fa9cc6306fe88fbd157487ccf192796e623bef1f898eb7659R86

[the remark this answers is not in the document]
Comment (marcoooo) [reply modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/FileDump_conf.pm:82]: No, I meant, do not use this `within` the pipeline, but put the string as a parameter to the pipeline. Something like "compara_host"

**Review by marcoooo — CHANGES_REQUESTED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**